test/test_helper.rb

```diff
@@ -12,14 +12,25 @@ def self.env
 end
 
 $adapter = ENV["ADAPTER"] || "postgresql"
-connection_options = {
-  adapter: $adapter,
-  database: "strong_migrations_test"
+
```

Comment:
Please don't make any changes to this file.

Reply:
We need to restore the connection back after the test - https://github.com/ankane/strong_migrations/pull/149/files#diff-c8ef59150a5fd973aeaded97986349c50366f7184bc49b7e847d43420496c146R110
How should I do that without the changes in this file?

Reply:
You can get the config with `ActiveRecord::Base.connection_db_config.configuration_hash` for Active Record 6.1+ and `ActiveRecord::Base.connection_config` for < 6.1.

Also, it looks like some of the tests are skipped for MySQL and MariaDB. `target_version` is supported for both of them, so it'd be good to include them (like the existing tests that use `target_version`).